core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -411,10 +411,12 @@ oppia.controller('EditorNavigation', [
     $scope.postTutorialHelpPopoverIsShown = false;
 
     $scope.$on('openPostTutorialHelpPopover', function() {
-      $scope.postTutorialHelpPopoverIsShown = true;
-      $timeout(function() {
-        $scope.postTutorialHelpPopoverIsShown = false;
-      }, 5000);
+      if ($(window).width() >= 1024) {
```

Comment:
Can you make use of windowDimensionsService here (in app.js) so that all our window-sizing stuff is in one place?